.appveyor.yml

```diff
@@ -0,0 +1,38 @@
+version: '{build}'
+
+# Build Using Our Hardware And Image
+build_cloud: "unwitting-accomplice Docker"
+image: "Ubuntu 18.04 Minimal Image 2"
```

Comment:
nit: Can we not add random version numbers to the image name?

Reply:
Removed, might need to go update the server tomorrow, we'll see if this builds.

Reply:
Ok, will need to go update the server tomorrow.